src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.service.impl.CsvReader;
+import core.basesyntax.service.impl.CsvWriter;
+import core.basesyntax.service.impl.FruitTransaction;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        String path = "src/main/java/core/basesyntax/Transactions.csv";
```

Comment:
1. Make this constant
2. Place the input and output files into the **src/main/resources folder.**
https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#place-the-input-and-output-files-into-the-srcmainresources-folder